Web/Startup.cs

```diff
@@ -36,22 +36,18 @@ public void ConfigureServices(IServiceCollection services)
 
             
             //Services
-            services.AddTransient<RecipeService>();
```

Comment:
When configuring Dependency Injection several options are available that determine the lifetime of the instance. By Transient every controller/service gets a new instance, that doesn't seem necessary for this use case. `AddSingleton` could be use to reuse instances. Checkout a more sophisticated description of the differences here:  https://stackoverflow.com/a/38139500/6310593